mcs/class/Makefile

```diff
@@ -105,6 +105,8 @@ monodroid_dirs := \
 	$(mobile_common_dirs)	\
 	Mono.CompilerServices.SymbolWriter	\
 	Mono.Btls.Interface	\
+	System.Runtime.Serialization.Formatters.Soap	\
+	System.Runtime.Remoting	\
```

Comment:
I am not sure this is good idea. IIRC XA ships whatever is in output folder so it'll hard to take this back